docs/admin/templates/extending-templates/prebuilt-workspaces.md

```diff
@@ -25,120 +34,120 @@ template parameters, Coder assigns an existing instance, reducing setup time sig
-that all unclaimed prebuilt workspaces are [owned](#workspace-ownership) by the `prebuilds` user, you may configure a quota for
-any group which this user appears in.
+The system always maintains the desired number of prebuilt workspaces for the active template version.
 
-Once the quota is exceeded, prebuilt workspaces will fail provisioning like regular workspaces would.
+### Relationship to workspace presets
 
-## Current Limitations
+Prebuilt workspaces are tightly integrated with [workspace presets](./parameters.md#workspace-presets-beta):
 
-- Organizations
+1. Each Prebuild is associated with a specific template preset.
+1. The preset must define all required parameters needed to build the workspace.
+1. Parameters that are not defined in the preset can still be customized by users when they claim a workspace.
+1. The preset parameters define the base configuration and are immutable after they're claimed.
+1. Prebuilds help presets deliver even faster workspace creation.
 
-   Prebuilt workspaces can only be utilized by the default organization.
+_Note: In future releases, we will allow operators to invalidate their prebuilt workspaces programmatically._
 
-   [coder/internal#364](https://github.com/coder/internal/issues/364)
+## Administration and troubleshooting
 
-- Autoscaling
+### Managing resource quotas
 
-   Prebuilt workspaces remain running until they are claimed.
-   We do not currently have an autoscaling mechanism to reduce the number of instances after working hours.
+Prebuilt workspaces can be used in conjunction with [resource quotas](../../users/quotas.md).
+Because unclaimed prebuilt workspaces are owned by the `prebuilds` user, you can:
 
-   [coder/internal#312](https://github.com/coder/internal/issues/312)
+1. Configure quotas for any group that includes this user.
+1. Set appropriate limits to balance Prebuild availability with resource constraints.
+1. Monitor quota utilization through Coder's dashboard.
 
-## Gotchas
+If a quota is exceeded, the prebuilt workspace will fail provisioning the same way other workspaces do.
 
-### Resource Replacement
+### Template configuration best practices
 
-When a prebuilt workspace is created, it is initially [owned](#workspace-ownership) by the `prebuilds` user and a random name
-is generated for it.
-When `terraform apply` runs, it will provide these values during provisioning in the
-[`coder_workspace`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace) and
-[`coder_workspace_owner`](https://registry.terraform.io/providers/coder/coder/latest/docs/data-sources/workspace_owner)
-datasources.
+#### Preventing resource replacement
 
-Once a prebuilt workspace is claimed, the ownership of that workspace changes to the requesting user and
-`terraform apply` is run again, now with updated values for the aforementioned datasources.
+When a prebuilt workspace is claimed, Terraform runs again with new values for the workspace owner and name.
```

Comment:
```suggestion
When a prebuilt workspace is claimed, another `terraform apply` run occurs with new values for the workspace owner and name.
```

Reply:
I feel this has gotten a lot less clear. How the new values for workspace owner and name are materially important here, and mentioning the `coder_` datasources is paramount.

Reply:
I'm trying to find a way to make this more active - does it make sense to say ?

```md
Coder runs `terraform apply` with new values...
```